CHANGES.md

```diff
@@ -34,7 +34,7 @@ namespace, under `cylc.jinja.filters`.
 Cylc Review was also removed in this version.
 
 -------------------------------------------------------------------------------
-## __cylc-8.0a2 (2019-Q4?)__
+## __cylc-8.0a2 (2020-07-??)__
```

Comment:
Bit of a delay :grimacing:

Reply:
Just a bit, still need to fill in the exact date though!